code/datums/status_effects/wound_effects.dm

```diff
@@ -205,7 +205,90 @@
 /datum/status_effect/wound/blunt/bone/critical
 	id = "compound"
 
+/datum/status_effect/wound/blunt/robotic/moderate
+	id = "unsecure_moderate"
+
+/datum/status_effect/wound/blunt/robotic/severe
+	id = "unsecure_severe"
+
+/datum/status_effect/wound/blunt/robotic/critical
+	id = "unsecure_critical"
+
+/datum/status_effect/wound/blunt/robotic/on_apply()
+	. = ..()
+	RegisterSignal(owner, COMSIG_MOB_SWAP_HANDS, PROC_REF(on_swap_hands))
+	on_swap_hands()
+
+/datum/status_effect/wound/blunt/robotic/on_remove()
+	. = ..()
+	UnregisterSignal(owner, COMSIG_MOB_SWAP_HANDS)
+	var/mob/living/carbon/wound_owner = owner
+	wound_owner.remove_actionspeed_modifier(/datum/actionspeed_modifier/robotic_blunt_wound)
+
+/datum/status_effect/wound/blunt/robotic/proc/on_swap_hands()
+	SIGNAL_HANDLER
+
+	var/mob/living/carbon/wound_owner = owner
+	if(wound_owner.get_active_hand() == linked_limb)
+		wound_owner.add_actionspeed_modifier(/datum/actionspeed_modifier/robotic_blunt_wound, (linked_wound.interaction_efficiency_penalty - 1))
+	else
+		wound_owner.remove_actionspeed_modifier(/datum/actionspeed_modifier/robotic_blunt_wound)
```

Comment:
Honestly, I really don't know - everything in this file is mostly just copypasted from wound code, since I just know it _works_

Im in the process of a 2nd wound refactor so I might take a second look at this and see if I can make it work better for the normal wounds and as a knock-on effect, fix this

Reply:
Oh also, this is just broken for bone wounds

So yeah probably it doesnt work for this either lol

Reply:
fixed in refactor